src/components/Tabber/PreviewModeSelector/PreviewModeSelector.components.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import Icon from '~components/Icon/Icon';
+
+import scss from './PreviewModeSelector.module.scss';
+
+import { PreviewModeProps } from './PreviewModeSelector.type';
+
+export function PreviewMode(props: PreviewModeProps): React.JSX.Element {
+  return (
+    <div key={props.item.id}>
+      <input
+        className={scss.previewModeInput}
+        id={props.item.id}
+        name="device"
+        type="radio"
+      />
+
+      <label className={scss.previewModeLabel} htmlFor={props.item.id}>
+        <span>{props.item.name}</span>
```

Comment:
nao entendi pq esse span existe visto que ele não é visível, é por acessibilidade? se for isso... não poderia ser um aria-label no input? se o span existe e nao é por acessibilidade, pode matar

Reply:
O span é pela acessibilidade, tem muita diferença um do outro?

Reply:
eu acho que seria bem melhor um aria-label, pois, no meu ponto de vista, é mais correto, pq no momento você tá com um span que não é exibido, sendo que o aria-label serve especificamente pra isso oq vc fez na mão, e é bom pra testes também, mas poderíamos ver com o @Alecell sobre isso